docs/fastdds/discovery/discovery_server.rst

```diff
@@ -79,9 +79,17 @@ In this architecture there are several key concepts to understand:
     since several *servers* can be hosted in the same machine, thus having the same IP, and also because multicast
     addresses are acceptable addresses.
 
-- A |SUPER_CLIENT| is a *client* that receives all the discovery information known by the *server*, in opposition to
+- A |SUPER_CLIENT| is a *client* that receives the discovery information known by the *server*, in opposition to
   *clients*, which only receive the information they need.
 
+  .. note::
+
+    A *Super_client* does not behave as a *Server* as it only receives the discovery information through the *Server* it
+    is connected to.
+    Also, the discovery information the *Server* relays concerns only about the endpoints (DataReaders and DataWriters)
+    that has been discovered in the discovery server network.
```

Comment:
The server does relay information about participants, but only if they have endpoints. IDK whether we even need this sentence

Reply:
I agree. With the first and latest sentence all the required information is already relayed.